File: 01-Assignment-02-02-2026/Using_Java/Program5.java
/*
Program 5
Java program to find the product of a set of real numbers
*/

import java.util.Scanner;

public class Program5 {

    public static void main(String[] args) {
        // setup scanner
        Scanner scanner = new Scanner(System.in);

        // get size
        System.out.print("Enter the number of real numbers : ");
        int size = scanner.nextInt();

        // define product variable
        double product = 1 ;

        for(int i = 0;i < size; i++){
            System.out.println("Enter the no. " + (i + 1) + " :");
            double num = scanner.nextDouble();
            product *=num;
        }

        // show result
        System.out.println("Product of given real numbers is : " + product);

        scanner.close();
    }

}

File: 01-Assignment-02-02-2026/Using_C/program_5.c
/* Program 5: Find the product of a set of real numbers */

#include <stdio.h>

int main() {
    int n ;
    printf("Total Numbers : ");
    scanf("%d",&n);

    if (n<1) {
        printf("Invalid Number");
        return 0 ;
    }

    float product = 1 ,temp;
    for (int i=0;i<n;i++){
        printf("Real No.(%d) : ",i+1);
        scanf("%f",&temp);
        product*=temp;
    }
    printf("Product of Given Numbers : %.2f",product);
    return 0;
}